pallets/messages/src/mock.rs

```diff
@@ -70,7 +72,10 @@ impl system::Config for Test {
 
 parameter_types! {
 	pub const MaxMessagesPerBlock: u32 = 500;
-	pub const MaxMessagePayloadSizeBytes: u32 = 100;
+	// In order to test max payload size for IPFS messages, this needs to be set
+	// to 1 byte less than the max IPFS payload (which would be using the largest
+	// hash size, SHA2-512). (We use SHA2-256 hashes for all other CIDs in the test suite).
+	pub const MaxMessagePayloadSizeBytes: u32 = 73;
```

Comment:
this change breaks the benchmark for `add_onchain_message` when ran as a test.
`cargo test --features runtime-benchmarks`

![Screenshot 2023-01-13 at 2 37 36 PM](https://user-images.githubusercontent.com/9152501/212431934-466d434a-4f2f-40f4-8792-5a4083649d9a.png)

Reply:
Fixed. Found a way to override MaxMessagePayloadSizeBytes for a single test.